flaky-tests/ci-providers/buildkite-quickstart.md

```diff
@@ -44,16 +47,16 @@ steps:
     key: tests
   - label: "Upload test results"
     commands:
-      - curl -fsSL --retry 3 "UPLOADER_LINK" -o ./trunk-analytics-uploader
-      - chmod +x ./trunk-analytics-uploader
-      - ./trunk-analytics-uploader upload --junit-paths *.xml --org-url-slug trunk --token $$TRUNK_TOKEN
+    - curl -fsSL --retry 3 "CLI_LINK" | tar -xvz > ./trunk-analytics-cli
+      - chmod +x ./trunk-analytics-cli
+      - ./trunk-analytics-cli upload --junit-paths *.xml --org-url-slug trunk --token $$TRUNK_TOKEN
```

Comment:
`--trunk-token` is not an option. You can double check this by following these docs and running `./trunk-analytics-cli upload --help`:

`Usage: trunk-analytics-cli upload [OPTIONS] --junit-paths <JUNIT_PATHS> --org-url-slug <ORG_URL_SLUG> --token <TOKEN>`

Reply:
I removed that extra `trunk` option.